cmd/kops/update_cluster.go

```diff
@@ -188,7 +189,11 @@ func RunUpdateCluster(f *util.Factory, cmd *cobra.Command, args []string, out io
 			fmt.Printf("\n")
 			fmt.Printf("Suggestions:\n")
 			fmt.Printf(" * list nodes: kubectl get nodes --show-labels\n")
-			fmt.Printf(" * ssh to the master: ssh -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
+			if cluster.Spec.Topology.Masters == kops.TopologyPublic {
+				fmt.Printf(" * ssh to the master: ssh -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
+			}else {
+				fmt.Printf(" * ssh to the bastion: ssh -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
+			}
```

Comment:
Yes that would work - but for the sake of keeping the two error messages decoupled I think it's fine to define the repeating text.

The Bastion has some work that will be coming in a future PR https://github.com/kubernetes/kops/issues/836 that will more than likely effect this string anyway.

Usually when writing go - I err on the side of decoupled functionality whenever possible :)